### PR TITLE
add exclude_automatic_assertions parameter to bioentityset/slimmer/

### PR DESCRIPTION
**Pull request by deepakunni3**
@nathandunn FYI

Query to test with:
```
https://api.monarchinitiative.org/api/bioentityset/slimmer/function?&slim=GO:0003824&slim=GO:0004872&slim=GO:0005102&slim=GO:0005215&slim=GO:0005198&slim=GO:0008092&slim=GO:0003677&slim=GO:0003723&slim=GO:0001071&slim=GO:0036094&slim=GO:0046872&slim=GO:0030246&slim=GO:0003674&slim=GO:0008283&slim=GO:0071840&slim=GO:0051179&slim=GO:0032502&slim=GO:0000003&slim=GO:0002376&slim=GO:0050877&slim=GO:0050896&slim=GO:0023052&slim=GO:0010467&slim=GO:0019538&slim=GO:0006259&slim=GO:0044281&slim=GO:0050789&slim=GO:0042592&slim=GO:0007610&slim=GO:0008150&slim=GO:0005576&slim=GO:0005737&slim=GO:0005856&slim=GO:0005739&slim=GO:0005634&slim=GO:0005694&slim=GO:0016020&slim=GO:0031982&slim=GO:0071944&slim=GO:0030054&slim=GO:0042995&slim=GO:0032991&slim=GO:0045202&slim=GO:0005575&subject=ZFIN:ZDB-GENE-000823-4
```

Will wait for our GOlr to come back online before testing.

### CODE REVIEW

File: biolink/api/entityset/endpoints/slimmer.py [hunk 1]
```diff
@@ -47,7 +48,6 @@ def get(self, category):
         results = map2slim(subjects=prots,
                            slim=slim,
                            rows=200,
-                           exclude_automatic_assertions=True,
```
Comment (nathandunn) [inline biolink/api/entityset/endpoints/slimmer.py:50]: per the call, should be `=False` but otherwise looks great.

**Review by nathandunn — APPROVED** [review]

File: biolink/api/entityset/endpoints/slimmer.py (hunk 1 above)
Comment (deepakunni3) [reply biolink/api/entityset/endpoints/slimmer.py:50]: The `exclude_automatic_assertions` is `False` by default, and it is passed in to `map2slim` via `args` dictionary.

Comment (nathandunn) [reply biolink/api/entityset/endpoints/slimmer.py:50]: gotcha

**Comment by deepakunni3** [comment]
@nathandunn Tested and the test query returns GO annotations that are IEA.

**Comment by nathandunn** [comment]
Awesome .. merge and push!  

Nathan

**Pull request merged**